charts/aws-efs-csi-driver/templates/node-daemonset.yaml

```diff
@@ -165,7 +168,7 @@ spec:
             type: Directory
         - name: efs-state-dir
           hostPath:
-            path: /var/run/efs
+            path: /var/run/efs-csi-driver
```

Comment:
why the path has been changed?

Reply:
So I thought this was really weird, it's an inconsistency between the Kustomize and the Helm Chart. If you look at `master`, the entry in the Helm Chart was added at some point 3 years ago and appears to have never changed. But the Kustomize was changed in PR #792, I assumed since the Kustomize was more up-to-date that was correct but if the Kustomize is incorrect then I'm happy to excise it.

Reply:
Not a problem. Will merge it